metrics/test_driver/common/process_manager/process_manager.dart

```diff
@@ -28,22 +29,28 @@ class ProcessManager {
   })  : _verbose = verbose,
         _quiet = quiet;
 
-  /// Kill all started processes and exit the app with the current [exitCode]
+  /// Kills all started processes and exit the app with the current [exitCode].
   void exitApp() {
     print("Cleaning...");
 
-    for (final pid in _startedPids) {
-      Process.killPid(pid);
+    for (final process in _startedProcesses) {
+      exitProcess(process);
     }
 
     exit(exitCode);
   }
 
+  /// Quites the process's application and kills the process.
+  void exitProcess(Process process) {
+    process.stdin.add(_quiteAppCommand);
```

Comment:
Do we send quit before killing it? I guess if we kill right away after sending quit command - then it probably not really doing much?

Reply:
We are sending the quit command and then killing the process because we want to be sure that the application running in process and the process itself are terminated properly. In case we won't send the quit command, the flutter application will be up on the web-server.

Reply:
Should this be renamed to `_quitAppCommand`? Also we might want to expand the comment to explain why we are sending the quit command. This way the reasoning is documented in the codebase.

quit = to exit
quite = to making less noise (typically used to silence logging output)

Reply:
@bootstraponline we discussed that as well offline - you should see improvements with new code.